### ISSUE TITLE
Make the unit bind arrow easier to follow

### ISSUE THREAD

**Issue by luihum** [Anuken/Mindustry-Suggestions#2666]
**Describe the content or mechanics you are proposing.**
When you click on a unit that's being controlled by a processor, a small yellow triangle appears pointing to the processor. 
When there's many processors and many units, such as in sandbox multiplayer, it becomes hard to follow and locate the processor. Maybe put a yellow dot at the direction the triangle points at, or make the pointing vertex a slightly darker shade of yellow.

**Describe how you think this content will improve the game. If you're proposing new content, mention how it may add more gameplay options or how it will fill a new niche.**

Besides improving accessibility, it helps debug processors, remove logic viruses, locate faulty processors, etc.

**Before making this issue, place an `X` in the boxes below to confirm that you have acknowledged them.** *Failure to do so may result in your request being closed automatically.*

1. - [X] I have checked the [Trello](https://trello.com/b/aE2tcUwF/mindustry-trello) to make sure my suggestion isn't planned or implemented in a development version.
2. - [X] I am familiar with all the content already in the game or have glanced at the wiki to make sure my suggestion doesn't exist in the game yet.
3. - [X] I have read `README.md` to make sure my idea is not listed under the "A few things you shouldn't suggest" category.

**Comment by OnyxFinger** [comment]
maybe a button to create a permanent line that tracks the unit ? Though that falls under the "do not suggest" few things from the readme

**Comment by Ineedause** [comment]
I would like to add to this, where on the unit description it says what process of the logic the unit is currently following show the x and y of the processor it is bound to

**Comment by oinkymoo** [comment]
> maybe a button to create a permanent line that tracks the unit ? Though that falls under the "do not suggest" few things from the readme

how would this be against the readme?
